packages/@contentful--app-scripts/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@contentful/app-scripts",
+  "version": "0.4.2",
+  "description": "A collection of scripts for building Contentful Apps",
+  "author": "Contentful GmbH",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/contentful/create-contentful-app.git",
+    "directory": "packages/app-scripts"
+  },
+  "engines": {
+    "node": ">=12 <15",
+    "npm": ">=6 <7"
+  },
+  "main": "lib/index.js",
+  "lint-staged": {
+    "*.{js}": [
+      "eslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "bin": {
+    "contentful-app-scripts": "bin/app-scripts"
+  },
+  "scripts": {
+    "prettier": "prettier **/*.js --write --ignore-path .gitignore",
+    "lint": "eslint ./lib",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "mocha ./lib/**/*.test.js",
+    "test:watch": "npm t -- --watch",
+    "pre-commit": "lint-staged",
+    "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+  },
+  "bugs": {
+    "url": "https://github.com/contentful/create-contentful-app/issues"
+  },
+  "homepage": "https://github.com/contentful/create-contentful-app#readme",
+  "directories": {
+    "lib": "lib",
+    "bin": "bin"
+  },
```

Comment:
🤷  where do these come from?

Reply:
Lerna 🙃 Also, it's compliant with [specification](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#directories) so I think it does not hurt

Reply:
Aaaah. I was looking at the npm 7 specs: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#directories
They removed all but `bin` and `man`.